src/Blueprint/Blueprint/Common/BlueprintClient.cs

```diff
@@ -207,6 +207,19 @@ public PSBlueprintAssignment CreateOrUpdateBlueprintAssignment(string subscripti
             return null;
         }
 
+        public PSWhoIsBlueprintContract GetBlueprintSpnObjectId(string scope, string assignmentName)
+        {
+            var result = blueprintManagementClient.Assignments
+                .WhoIsBlueprintWithHttpMessagesAsync(scope, assignmentName).GetAwaiter().GetResult();
```

Comment:
@filizt minor: any reason you aren't using `WhoIsBlueprint()` here?

Reply:
Done.